patches/chromium/chore_modify_chromium_handling_of_mouse_events.patch

```diff
@@ -75,25 +75,26 @@ index 7f29d902ae0a2f980a56e77e6e25935dab3c0685..ab35b4e2cd2f71ddd6d2fe34319b4b62
 +                    static_cast<DWORD>(message_time),
 +                    {CR_GET_X_LPARAM(l_param), CR_GET_Y_LPARAM(l_param)}};
 +  ui::MouseEvent event(msg);
++  bool right_click_handled = false;
    if (message == WM_RBUTTONUP && is_right_mouse_pressed_on_caption_) {
      // TODO(pkasting): Maybe handle this in DesktopWindowTreeHostWin, where we
      // handle alt-space, or in the frame itself.
      is_right_mouse_pressed_on_caption_ = false;
      ReleaseCapture();
 +
-+    bool handled = delegate_->HandleMouseEvent(&event);
++    right_click_handled = delegate_->HandleMouseEvent(&event);
```

Comment:
Any reason to handle it before the hittest  and not something like,

```
if (w_param == HTCAPTION || w_param == HTSYSMENU) {
  handled = delegate_->HandleMouseEvent(&event);
  if (!handled) {
    ShowSystemMenuAtScreenPixelLocation(hwnd(), gfx::Point(screen_point));
  }
  return 0;
```

Reply:
no, not really - i tweaked it.